csharp/ql/lib/semmle/code/csharp/dataflow/CsvValidation.qll

```diff
@@ -0,0 +1,140 @@
+/** Provides a query predicate to check the CSV data for validation errors. */
+
+import csharp
```

Comment:
As discussed offline, let's move this back into `ExternalFlow.qll`.

Reply:
Sure thing - I will get on it!